scripts/migrations.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+for MIGRATION in $(ls metadata_store_scripts/V*.sql)
+do
+  git checkout origin/main -- metadata_store_scripts/bootstrap.sql
+  script_version_and_comment=( $(basename -- "$MIGRATION" .sql ) )
+  echo "$script_version_and_comment"
+  lines_in_bootstrap=( $(grep "$script_version_and_comment" metadata_store_scripts/bootstrap.sql | wc -l) )
+  if [[ $lines_in_bootstrap -eq 2 ]]; then
+    # Nothing to do, assume this is correct
+    true
+  elif [[ $lines_in_bootstrap -eq 0 ]]; then
+    sed "s/\-\- The contents of each of those files follows/-- * $script_version_and_comment\n-- The contents of each of those files follows/g" metadata_store_scripts/bootstrap.sql > metadata_store_scripts/bootstrap.sql.tmp
```

Comment:
Can we add this following echo message: `echo "Adding $script_version_and_comment script to the bootstrap.sql file..."`

followed by this comment: `# Insert $script_version_and_comment before the placeholder comment "-- The contents of each of those files follows in bootstrap.sql"`